viritin/src/main/java/org/vaadin/viritin/fields/LabelField.java

```diff
@@ -58,7 +58,7 @@ public LabelField(String caption) {
     }
 
     private CaptionGenerator<T> captionGenerator = new ToStringCaptionGenerator<>();
-    private Label label = new RichText();
+    protected Label label = new RichText();
```

Comment:
If we override `getLabel`in `HeaderField` we don't have to expose the label propertie